### ISSUE TITLE
Feature: list existing albums

### ISSUE THREAD

**Issue by cconde** [mrworf/photoframe#149]
I'm being unable to type properly the album name (despite making copy-paste from Google Photos name)

I'm sure I'm doing something wrong but it would be really interesting to have some way of listing the names of the albums to grab the proper name. A link on each name to get it added inmetiately would be great.

Hope this helps,
Thank you for your wonderful job!!!

**Comment by cconde** [comment]
BTW: after two or three authorization (first of one was with the API disabled) I've got it working.

I think the album listing is still a nice feature, and I would also suggest a "Disabled API" notification

Thanks again!!!

**Comment by mrworf** [comment]
Good idea with the API disabled thing. Listing the albums will happen once I allow providers to have their own UI (but that's still ways away unfortunately)
